transaction-view/src/static_account_keys_meta.rs

```diff
@@ -10,7 +10,7 @@ use {
 // This means the maximum number of 32 byte keys is 38.
 // 38 as an min-sized encoded u16 is 1 byte.
 // We can simply read this byte, if it's >38 we can return None.
-const MAX_STATIC_ACCOUNTS_PER_PACKET: u8 =
+pub const MAX_STATIC_ACCOUNTS_PER_PACKET: u8 =
```

Comment:
can't comment below, if this module is now `pub` - probably want the `StaticAccountKeysMeta` struct to be `pub(crate)` instead.

Reply:
b07ddccc51c67613ffa965e9c6e23f3882124a0e